core/src/OpenMPTarget/Kokkos_OpenMPTarget_Exec.hpp

```diff
@@ -867,11 +857,11 @@ class OpenMPTargetExecTeamMember {
 
   template <class ValueType, class JoinOp>
   KOKKOS_INLINE_FUNCTION ValueType team_reduce(const ValueType& value,
-                                               const JoinOp& op_in) const {
+                                               const JoinOp&) const {
 #pragma omp barrier
 
     using value_type = ValueType;
-    const JoinLambdaAdapter<value_type, JoinOp> op(op_in);
+    //    const JoinLambdaAdapter<value_type, JoinOp> op(op_in);
```

Comment:
Why leave this commented in? Was this a bug that `op` was never used below? I don't see the team_reduce here being restricted to +?

Reply:
On a second glace, I'm not quite sure if `OpenMPTarget` is even implementing the right interface here. All the other backends only seem to implement
```C++
template <typename ReducerType>
KOKKOS_INLINE_FUNCTION
typename std::enable_if<is_reducer<ReducerType>::value>::type
team_reduce(ReducerType const& reducer) const noexcept;

template <typename ReducerType>
KOKKOS_INLINE_FUNCTION
typename std::enable_if<is_reducer<ReducerType>::value>::type
team_reduce(ReducerType const& reducer,
            typename ReducerType::value_type& value) const noexcept;
```
(HPX has a dummy implementation and there is an implementation for `Threads`, though).